app/views/charge.njk

```diff
@@ -349,7 +349,12 @@
                     {{ __p("commonConjunctions.or") }}
                   </span>
                   <img src="/images/amex-security-code.png" class="amex-cvc hidden" alt="{{ __p("cardDetails.amexcvcTip") }}"/>
-                </div>
+                 </div>
+                   {% if agreementId %}
```

Comment:
I believe the spec calls for `savePaymentInstrumentToAgreement` flag to be serialised and explicitly checked to show this (placholder) disclaimer. 

For the existing pattern just having an agreement ID might be enough to know this is a payment to set up an agreement but I would be wary of of relying on that logic, especially as this is extended in the future.

To make this explicit this check should be for

```njk
{% if savePaymentInstrumentToAgreement and agreementId %}
```

This will just be for charges consumed by the card frontend (not available through the API, for example)